src/pages/api/upload-background.ts

```
@@ -0,0 +1,60 @@
+      return res.status(400).json({
+        error: 'Invalid file type',
+        message: 'Only JPG, PNG, GIF and WebP images can be uploaded',
+      })
+    }
+
+    const bgDir = path.join(process.cwd(), 'public/backgrounds')
+    if (!fs.existsSync(bgDir)) {
+      fs.mkdirSync(bgDir, { recursive: true })
+    }
+
+    const newPath = path.join(bgDir, file.originalFilename || 'background' + extension)
```

ESLint

[error] Replace `bgDir,·file.originalFilename·||·'background'·+·extension` with `⏎······bgDir,⏎······file.originalFilename·||·'background'·+·extension⏎····`

(prettier/prettier)